pytorch_lightning/utilities/distributed.py

```diff
@@ -203,10 +204,13 @@ def all_gather_ddp_if_available(
     Return:
         A tensor of shape (world_size, batch, ...)
     """
+    if group is None:
+        group = torch.distributed.group.WORLD
```

Comment:
same change as this PR https://github.com/PyTorchLightning/pytorch-lightning/pull/5125